packages/netlify-cms-core/src/components/Collection/Entries/Entries.js

```diff
@@ -13,34 +24,40 @@ const Entries = ({
   cursor,
   handleCursorActions,
   t,
+  page,
 }) => {
   const loadingMessages = [
     t('collection.entries.loadingEntries'),
     t('collection.entries.cachingEntries'),
     t('collection.entries.longerLoading'),
   ];
 
-  if (entries) {
-    return (
-      <EntryListing
-        collections={collections}
-        entries={entries}
-        viewStyle={viewStyle}
-        cursor={cursor}
-        handleCursorActions={handleCursorActions}
-      />
-    );
+  if (isFetching && page === undefined) {
+    return <Loader active>{loadingMessages}</Loader>;
   }
 
-  if (isFetching) {
-    return <Loader active>{loadingMessages}</Loader>;
```

Comment:
We should drop that "this could take several minutes" message entirely, it's so disheartening to see. It's rarely true but shows right away if we take more than 10 seconds.

Might be worth doing really soon as more folks will see it with sorting live.